src/roles/wildchild.py

```diff
@@ -1,78 +1,69 @@
-import random
 import re
+import random
+import itertools
+import math
+from collections import defaultdict
 
-import src.settings as var
+import botconfig
 from src.utilities import *
-from src import users, channels, debuglog, errlog, plog
-from src.functions import get_players, get_all_players, get_main_role
-from src.decorators import cmd, event_listener
+from src import channels, users, debuglog, errlog, plog
+from src.functions import get_players, get_all_players, get_main_role, get_reveal_role, get_target
+from src.decorators import command, event_listener
 from src.containers import UserList, UserSet, UserDict, DefaultUserDict
 from src.messages import messages
 from src.events import Event
 
-WILD_CHILDREN = set()
-IDOLS = {}
+WILD_CHILDREN = UserSet()
+IDOLS = UserDict()
 
-@cmd("choose", chan=False, pm=True, playing=True, phases=("night",), roles=("wild child",))
-def choose_idol(cli, nick, chan, rest):
+@command("choose", chan=False, pm=True, playing=True, silenced=False, phases=("night",), roles=("wild child",))
```

Comment:
silenced=False is the default, so this addition isn't really needed.

Reply:
Alright.